Cargo.toml

```diff
@@ -185,4 +186,5 @@ ya-utils-actix = { path = "utils/actix_utils"}
 ya-utils-futures = { path = "utils/futures"}
 ya-utils-path = { path = "utils/path"}
 ya-utils-process = { path = "utils/process"}
+ya-diesel-utils = { path = "utils/diesel-utils"}
```

Comment:
you should update also ya-client to have `/agreementEvents` instead `/agreements/events`

Reply:
Will do in #730